e2e/live/end-to-end-test.sh

```diff
@@ -439,11 +439,11 @@ ${BIN_DIR}/kpt live apply e2e/live/testdata/rg-test-case-1a > $OUTPUT_DIR/status
 # The ResourceGroup inventory CRD is automatically installed on the initial apply.
 assertContains "installing inventory ResourceGroup CRD"
 assertContains "namespace/rg-test-namespace"
+assertContains "1 resource(s) applied. 0 created, 1 unchanged, 0 configured, 0 failed"
 assertContains "pod/pod-a created"
 assertContains "pod/pod-b created"
 assertContains "pod/pod-c created"
 assertContains "4 resource(s) applied. 3 created, 1 unchanged, 0 configured, 0 failed"
-assertContains "0 resource(s) pruned, 0 skipped, 0 failed"
```

Comment:
so `prune` summary wasn't useful ? It will be comforting to know if any resources were pruned.

Reply:
We are changing the output so it only shows up if anything is actually pruned. The current output has confusing for users since the context of the summary output is not clear.